app/client/components/cancel/gw/gwCancellationFlowStart.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { trackEvent } from "../../analytics";
+import { PageContainerSection } from "../../page";
+import { hrefStyle } from "../cancellationConstants";
+
+const trackCancellationClickEvent = (eventLabel: string) => () =>
+  trackEvent({
+    eventCategory: "cancellation",
+    eventAction: "click",
+    eventLabel
+  });
+
+export const gwCancellationFlowStart = () => (
+  <PageContainerSection>
+    <h3>
+      We’re sorry to hear you’re thinking of cancelling your Guardian Weekly
+      subscription.
+    </h3>
+
+    <p>
+      Your support means The Guardian can remain editorially independent, free
+      from the influence of billionaire owners and politicians. This enables us
+      to give a voice to the voiceless, challenge the powerful and hold them to
+      account. Our readers’ support helps us to keep our journalism free of a
+      paywall, so it’s open and accessible to all.
+    </p>
+
+    <p>
+      If you’re looking to take a break- it’s possible to take a suspension from
```

Comment:
Do we want the dash in this sentence?

Reply:
@JessL123 is this a typo in the copy document?

Reply:
Perhaps change to a comma ?

Reply:
done - https://github.com/guardian/manage-frontend/pull/391/commits/b97e5f9363a794fce2d75571ca70c30758c86316